<p align="right">
  in conjunction with
  <a href="http://ieeebibm.org/BIBM2021/" target="_blank">IEEE BIBM 2021</a>
</p>


<p align="right">Houston, TX, USA</p>

<p align="right">December 9-12, 2021</p>

# Description

With the advances of sequencing techniques, single cell sequencing opens an unprecedented opportunity for multiomics analysis at single-cell resolution. It has allowed us to simultaneously profile multiomic layers of DNA, RNA, protein, methylation, or open chromatin nucleosome positioning in single cell to analyze the underlying mechanisms of human diseases and cellular development. To date, many algorithms have been designed to analyze different types of single cell data, and a majority of them have achieved great success in dealing with problems such as sequencing artifacts, grouping cells into cell types, and inferring cell trajectory. However, most of the approaches are data-dependent and have their own pros and cons in dealing with different experimental design and data types. **Development of algorithms and tools for integrative analysis of multiomic single-cell profiling is a crucial step toward understanding the interplay of biological systems.**

We invite investigators to contribute Original Research on computational methods in analyzing single cell multiomic data, and their applications in molecular biological and complex diseases. Potential topics include but are not limited to the following:

- Computational models for integrating multiomic single-cell profiling, e.g. scRNA-seq, scATAC-seq, scCLIP-seq, single-cell spatial transcriptomics.

- Novel analyses to identify risk factors of cancers or complex genetic diseases in single-cell resolution

- Bioinformatics software and databases for analyzing single-cell data.

- Imputation and cell type clustering on different types of single-cell data

- New algorithms or tools to analyze copy number variation on scRNA-seq data

- Trajectory inference

- Computational tools to generate synthetic single-cell data

- New methods to reconstruct gene regulatory network

- Algorithms to integrate single-cell and bulk sequencing data

# Important Dates (Tentative)

- **Oct. 20, 2021**: Due date for full workshop papers submission

- **Nov. 5, 2021**: Notification of paper acceptance to authors

- **Nov. 21, 2021**: Camera-ready of accepted papers

- **Dec. 11, 2021**: Workshops

# Program Chairs

- **Eric Lu Zhang**, Department of Computer Science, Hong Kong Baptist University

- **Xin (Maizie) Zhou**, Department of Biomedical Engineering and Computer Science, Vanderbilt University

- **Xian Fan Mallory**, Department of Computer Science, Florida State University

- **Debajyoti Chowdhury**, School of Chinese Medicine, Hong Kong Baptist University

# Program Committee Members

- **Ruli Gao**, Houston Methodist Hospital

- **Wenji Ma**, Chinese Academy of Science

- **Yong Fei Wang**, The University of Hong Kong

- **Wenjun Shen**, Shantou University

- **Zheng Hu**, University of Chinese Academy of Sciences

- **Ye Yuan**, Shanghai Jiao Tong University

# KEYNOTE SPEAKERS

- **Wei Vivian Li**，Assistant Professor, Department of Biostatistics and Epidemiology， 
Rutgers School of Public Health, Rutgers university

- **Ye Yuan**, Associate Professor , Department of Automation,  Shanghai Jiao Tong University

# Workshop Submission Requirement

Please submit a full-length paper (up to <font color='red'> 8 page IEEE 2-column format </font>) through the online submission system (you can download the format instruction [here](http://www.ieee.org/conferences_events/conferences/publishing/templates.html).
Electronic submissions (in PDF or Postscript format) are required. Selected participants will be asked to submit their revised papers in a format to be specified at the time of acceptance.  

# Workshop Online Submission:

**Submit a paper** [Click here](https://wi-lab.com/cyberchair/2021/bibm21/scripts/submit.php?subarea=S09&undisplay_detail=1&wh=/cyberchair/2021/bibm21/scripts/ws_submit.php
)
